### ISSUE TITLE
Using selenium-wire with undetected chrimedriver

### ISSUE THREAD

**Issue by Chris230291** [wkeeling/selenium-wire#366]
Hi I was wondering if someone could help me understand how to use selenium-wire with the undetected chromedriver.
More specifically the interceptor.

```
#! /usr/bin/env python3
import undetected_chromedriver.v2 as uc
from seleniumwire import webdriver
from seleniumwire.undetected_chromedriver.v2 import Chrome, ChromeOptions

def interceptor(request, response):
    print (request.url)

options = uc.ChromeOptions()
#options.add_argument('--headless')
driver = uc.Chrome(options=options)
driver.response_interceptor = interceptor
driver.get('https://github.com/wkeeling/selenium-wire')
```

Can anyone see where I am going wrong?
Chris

**Comment by wkeeling** [comment]
Thanks for the issue. It looks like you've imported undetected_chromedriver directly. You can omit this import and just use the version from Selenium Wire directly. 

```

#! /usr/bin/env python3
from seleniumwire import webdriver
from seleniumwire.undetected_chromedriver.v2 import Chrome, ChromeOptions

def interceptor(request, response):
    print (request.url)

options = ChromeOptions()
#options.add_argument('--headless')
driver = Chrome(options=options)
driver.response_interceptor = interceptor
driver.get('https://github.com/wkeeling/selenium-wire')

```

**Comment by Chris230291** [comment]
Thanks for the reply. It is working now.
Unfortunately it doesn't solve my problem though.
I used to be able to load [this ](https://www.virginmediatelevision.ie/player/live/virgin-media-one/)site with selenium wire and find the streaming video, but now the site does not load at all. I figured the site must have been updated to detect bots or whatever. That's when I came across the undetected driver. Perhaps the issue is something else though?

Chris

**Comment by wkeeling** [comment]
The URL https://www.virginmediatelevision.ie/player/live/virgin-media-one/ seems to load OK for me when I run undetected chromedriver v2. I can see the rendered page. 

When you say the site does not load, are you seeing an error or something else?

**Comment by Chris230291** [comment]
For me, when using the code posted above,  the browser loads but the screen is grey...

![image](https://user-images.githubusercontent.com/5328818/129425568-c430652f-3666-4186-97c3-90f67a658c19.png)

**Comment by wkeeling** [comment]
Discovered that this is being caused by a bug in the response interceptor functionality. It's being addressed by #378 . In the meantime, you should be able to workaround the issue by passing the `disable_encoding` option in the seleniumwire_options:

```python
driver = webdriver.Chrome(seleniumwire_options={'disable_encoding': True})
```

**Comment by Chris230291** [comment]
Thanks.

**Comment by wkeeling** [comment]
Fixed in 4.5.2
